app/src/main/java/org/simple/clinic/bp/entry/confirmremovebloodpressure/ConfirmRemoveBloodPressureEffectHandler.kt

```diff
@@ -33,9 +33,9 @@ class ConfirmRemoveBloodPressureEffectHandler @AssistedInject constructor(
       effects
           .observeOn(schedulersProvider.io())
           .switchMap { bloodPressureRepository.measurement(it.bloodPressureMeasurementUuid) }
-          .doOnNext { bloodPressureRepository.markBloodPressureAsDeleted(it) }
           .switchMap { bloodPressureMeasurement ->
-            patientRepository.updateRecordedAt(bloodPressureMeasurement.patientUuid)
+            bloodPressureRepository.markBloodPressureAsDeleted(bloodPressureMeasurement)
```

Comment:
Was this old returns a `Completable` that is never subscribed to problem? 😅

Reply:
Yep, I thought it was a blocking call. Since tests were working everything seemed fine 😓

Reply:
Yeah, this is a problem with mocks since with the mock we only verify that the method was invoked, not that the `Completable` it returned was subscribed to.